storio-content-resolver-annotations/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/StorIOContentResolverType.java

```diff
@@ -0,0 +1,22 @@
+package com.pushtorefresh.storio.contentresolver.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Annotation for marking class as StorIOContentResolverType to generate object mapping code
+ */
+@Target(TYPE)
+@Retention(RUNTIME) // we allow users to write reflection based code to work with annotation
+public @interface StorIOContentResolverType {
```

Comment:
I think we should also add ability to set different `uris` for `query`, `insert`, `update`, `delete`, I know several projects that use different uris for different operations :crying_cat_face: